packages/xs/ezxenstore.0.1.0/url

```diff
@@ -0,0 +1,2 @@
+https://github.com/xapi-project/ezxenstore/archive/v0.1.0/ezxenstore-0.1.0.tar.gz
```

Comment:
[ERROR] File "/home/opam/.opam/repo/xs-opam/packages/xs/ezxenstore.0.1.0/url", line 1, character 6-7: '/' is not a valid token.
Fatal error:
OpamTypesBase.Lexer_error("'/' is not a valid token")